scene/animation/tween.cpp

```diff
@@ -45,6 +45,7 @@ Tween::interpolater Tween::interpolaters[Tween::TRANS_MAX][Tween::EASE_MAX] = {
 	{ &circ::in, &circ::out, &circ::in_out, &circ::out_in },
 	{ &bounce::in, &bounce::out, &bounce::in_out, &bounce::out_in },
 	{ &back::in, &back::out, &back::in_out, &back::out_in },
+	{ &spring::out, &spring::out, &spring::out, &spring::out }, // Spring is the same for each easing.
```

Comment:
As I said above, I think this comment is probably a wrong. It should be possible to implement in/out/inout/outin the same way as Bounce.

Reply:
Not sure what you mean by "the same as Bounce", but in the [previous PR](https://github.com/godotengine/godot/pull/58999) it was settled that it's ok if spring implements single easing IF it's documented. Not sure if bounce fits here this way.